privacyidea/api/lib/prepolicy.py

```diff
@@ -991,6 +991,58 @@ def check_anonymous_user(request=None, action=None):
     return True
 
 
+def check_admin_tokenlist(request=None, action=None):
+    """
+    Depending on the policy scope=admin, action=tokenlist, the
+    allowed_realms parameter is set to define, the token of which
```

Comment:
Hm, I think since we're now calling the parameter ``pi_allowed_realms``, we should also use this name in the docstring. Could you adjust the docstring accordingly?

Reply:
will do